requirements.in

```diff
@@ -1,6 +1,6 @@
-Flask>=1.0.4,<1.1.0
-Flask-Login>=0.4.1,<0.5.0
-Flask-WTF>=0.14.2,<0.15.0
+Flask==1.0.4 # pyup: >=1.0.0,<1.1.0
+Flask-Login==0.5.0
```

Comment:
Do we want to upgrade Flask-Login to 0.5.0? The other apps are all still on 0.4.1...

Reply:
We should pin this to 0.4.1 as per https://trello.com/c/2GcrMr2g/401-ensure-all-repos-have-pip-and-pip-tools-setup-similarly